std/stdio.d

```diff
@@ -2772,6 +2772,14 @@ See $(LREF byChunk) for an example.
         return LockingTextWriter(this);
     }
 
+    @system unittest
+    {
+        // ensure that lockingTextWriter cannot accidentally accept ubyte arrays
```

Comment:
How about including a link to the issue for additional infos?
(might be helpful to someone in the future)